model/pluginconf.go

```diff
@@ -0,0 +1,38 @@
+package model
+
+// PluginConf holds information about the plugin
+type PluginConf struct {
+	ID       uint `gorm:"primary_key;AUTO_INCREMENT;index"`
+	UserID   uint
+	FileName string
+	Token    string `gorm:"unique_index"`
+	Enabled  bool
+	Config   []byte
+}
+
+// PluginConfExternal Model
+//
+// Holds information about a plugin instance for one user.
+//
+// swagger:model PluginConf
+type PluginConfExternal struct {
+	// The user id.
+	//
+	// read only: true
+	// required: true
+	// example: 25
+	UserID uint `json:"user_id"`
+	// The user name. For login.
+	//
+	// required: true
+	// example: P1234
+	Token string `binding:"required" json:"token" query:"token" form:"token"`
+	// The file name of the plugin.
+	//
```

Comment:
Should the filename be editable, I see no usage why a user should be able to do this.
So add this here:
```
// read only: true
// required: true
```